src/RollbarJsHelper.php

```diff
@@ -63,7 +63,10 @@ public function addJs($headers = null, $nonce = null, $customJs = "")
      */
     public function configJsTag()
     {
-        return "var _rollbarConfig = " . json_encode($this->config, JSON_FORCE_OBJECT) . ";";
+        return "var _rollbarConfig = " . json_encode(
+            $this->config,
+            $this->config ? null : JSON_FORCE_OBJECT
+        ) . ";";
```

Comment:
I think the intent of the `JSON_FORCE_OBJECT` was to ensure that the highest level was always an object, even when given as an empty array: eg, `"{}" === (new \Rollbar\RollbarJsConfig([]))->configJsTag()`. Given that, I think it'd be more clear to implement as:

```
return "var _rollbarConfig = " . json_encode((object)$this->config);
```

That ensures the top-most result is a JSON object, while preserving nested PHP arrays.